data/categories.txt

```diff
@@ -12952,39 +12954,45 @@ fa:لوازم یدکی
 mr:कारचे भाग|गाडीचे स्पेर पार्ट
 
 tourism-chalet|@category_hotel
-en:4Chalet|5holiday cottage|vacation home
-ru:3Шале|коттедж|дом отдыха
-bg:Хижа|3Шале|туризъм
-ar:شاليه
-cs:4Chalet|Horská chata
-da:Hytte
-nl:4Chalet
-fi:Mökki
-fr:4Chalet|Meublé de tourisme
-de:Hütte
-hi:3षाले
-hu:Nyaralóház|Kulcsosház
-id:4Chalet
-it:4Chalet
-ja:シャレー|山小屋
-ko:샬레|방갈로|산장
-nb:4Chalet
+en:5Holiday Cottage|5vacation home
+ar:عطلة المنزلية
+be:Катэдж для адпачынку
+bg:Ваканционна вила
+ca:Casa rural de vacances
+cs:Rekreační chata
+da:Feriehus
+de:Ferienhäuschen
+el:Εξοχικό σπίτι διακοπών
+es:Casa de vacaciones
+es-MX:Cabaña de vacaciones
+et:Puhkemaja
+eu:Oporretako Etxea
+fa:کلبه تعطیلات
+fi:Lomamökki
+fr:Chalet de vacances
+he:קוטג' נופש
+hi:हॉलिडे कॉटेज
+hu:Nyaraló házikó
+id:Pondok Liburan
+it:Cottage per le vacanze
+ja:ホリデー・コテージ
+ko:홀리데이 코티지
+mr:हॉलिडे कॉटेज
+nb:Feriehus
+nl:Vakantiehuis
 pl:Domek letniskowy
-pt:4Chalé
-pt-BR:4Chalé
-ro:4Cabană
-es:4Chalé
-et:Suvila
-eu:4Txaleta
-sv:Stuga
-th:ชาเลต์
-tr:Dağ evi
-uk:3Шале
-zh-Hans:木屋
-zh-Hant:木屋
-el:Σαλέ
-sk:4Chata
-fa:کلبه
+pt:Casa de férias
+ro:Cabană de vacanță
+ru:Коттедж для отдыха|дом отдыха|турбаза|база отдыха
```

Comment:
Isn't "chalet" mean premium cottage, rather than "турбаза"?

Reply:
Its not necessary premium, but should have self-contained cooking and bathroom facilities.
So actually you're right because "турбаза" if often with a common eating area i.e. houses are not necessary self-contained.
Removed!